pytorch_lightning/core/lightning.py

```diff
@@ -1447,18 +1447,20 @@ def training_step(...):
         self._verify_is_manual_optimization('manual_backward')
 
         # backward
-        self.trainer.fit_loop.epoch_loop.batch_loop.backward(loss, optimizer=None, opt_idx=None, *args, **kwargs)
+        self.trainer.fit_loop.epoch_loop.batch_loop.backward(loss, None, None, *args, **kwargs)
```

Comment:
the `batch_loop.backward` method has these as positional, so we pass them like that

Reply:
but the previous way was much clearer to read and Python is totally fine if you name the positional args.